packages/volto/src/components/manage/Blocks/Block/EditBlockWrapper.jsx

```diff
@@ -105,15 +116,50 @@ const EditBlockWrapper = (props) => {
         <div className={`ui drag block inner ${type}`}>
           {children}
           {selected && !required && editable && (
-            <Button
-              icon
-              basic
-              onClick={() => onDeleteBlock(block, true)}
-              className="delete-button"
-              aria-label={intl.formatMessage(messages.delete)}
-            >
-              <Icon name={trashSVG} size="18px" />
-            </Button>
+            <div class="absolute-div">
```

Comment:
i think it's better to change class name with a more specific name for this case